collect_app/src/main/res/values/colors.xml

```diff
@@ -25,7 +25,7 @@ the License.
     <!-- Light Theme Palette -->
     <color name="lightPrimaryTextColor">@android:color/black</color>
     <color name="lightIconColor">@android:color/black</color>
+    <color name="card_ui_activity_background">#f5f5f5</color>
```

Comment:
Yes, that's a good idea.

Reply:
@lognaturel where do I find `colorBackground`? Searched the android colors but didn't find a reference to it.

Reply:
I wrote this in Slack but for the sake of completeness -- just take out the special case and the background will use the theme default of #FAFAFA. I believe it may still be white on older versions of Android but that's probably ok. An option would be to force it at the theme configuration level so it's consistent.

Reply:
Okay! Removed the check.